.github/workflows/php-tests.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         with:
           php-version: ${{ matrix.php-versions }}
           tools: cs2pr, phpcs, phpstan
+          extensions: uopz
```

Comment:
https://github.com/marketplace/actions/setup-php-action#pcov should probably define this too